evadb/third_party/databases/postgres/requirements.txt

```diff
@@ -0,0 +1,2 @@
+psycopg<=2.9.7
+pandas<=2.0.2
```

Comment:
Is there reason we want to have a cap on the version? This very likely leads to version conflicts, given postgres is not installed with the evadb.  From my experience with the current setup.py, `pip install -e .[dev,postgres]` works, but `pip install -e .[dev]` and then `pip install -e .[postgres]` will print out version conflict warnings/errors.

Reply:
I see. Removed it.